src/index.html

```diff
@@ -1,14 +1,33 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <meta http-equiv="X-UA-Compatible" content="ie=edge">
-  <title>Enclosures</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Enclosures</h1>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Enclosures</title>
+    <link rel="stylesheet" href="./style.css">
+  </head>
+  <body>
+    <div class="blue-box bordered">
+      <div class="yellow-box bordered">
+        <p id="qa-block" class="grey-box bordered">
+          In programming languages, a closure is a technique for implementing lexically scoped name binding in a language with first-class functions. Operationally, a closure is a record storing a function together with an environment.
+        </p>
+      </div>
+    </div>
+    <div class="blue-box bordered">
+      <div class="yellow-box bordered">
+        <p class="grey-box bordered">
+          У програмуванні замиканням називають підпрограму, що виконується в середовищі, яке містить одну або більше зв’язаних змінних. Підпрограма має доступ до цих змінних під час виконання.
```

Comment:
watch today's lecture till minute 8 and set the right formatting to your VScode, such long text should be wrapped to the new line
https://www.youtube.com/watch?v=ScTyxXA8SA8&list=PL7FuXFaDeEX0sQ0Zhwk2ixfwS1P6e0iuW&index=5